Core/GDCore/Extensions/Builtin/AudioExtension.cpp

```diff
@@ -146,69 +146,83 @@ void GD_CORE_API BuiltinExtensionsImplementer::ImplementsAudioExtension(
       .AddAction("ModVolumeSoundCanal",
                  _("Volume of the sound on a channel"),
                  _("This action modifies the volume of the sound on the "
-                   "specified channel. The volume is between 0 and 100."),
+                   "specified channel."),
```

Comment:
Just to confirm.
This feels right removing it from here and having it in the placeholder, but what if someone enter 150, is there an error message reminding them that this is not ok? (because the placeholder disappears)

Reply:
No, there is no error. I think it could be something similar to UnitMeasurement, something like AcceptableRange that should be linked to a parameter to enforce that

Reply:
I guess the placeholder doesn't disappear. It's still on the top with a smaller font.

Reply:
ah nice! I thought for a moment it disappeared. All good then